modules/xpfunmac.cmake

```diff
@@ -735,14 +735,6 @@ function(xpInstallPro)
   include(CPack)
 endfunction()
 
-function(xpListPrependToAll var prefix)
-  set(listVar)
-  foreach(f ${ARGN})
-    list(APPEND listVar "${prefix}/${f}")
-  endforeach()
-  set(${var} "${listVar}" PARENT_SCOPE)
-endfunction()
-
```

Comment:
I don't know what your policy is on backwards compatibility.  I can put these lines back in if desired.